src/main/scala/nl.knaw.dans.easy.bagindex/server/BagIndexServletComponent.scala

```diff
@@ -162,6 +152,7 @@ trait BagIndexServletComponent {
     private def defaultErrorHandling(t: Throwable): ActionResult = {
       t match {
         case e: IllegalArgumentException => BadRequest(e.getMessage)
```

Comment:
With the replacement of `IllegalArgumentException` above with `UUIDError`, is this pattern match still necessary? Not sure...

Reply:
Yes, it is still needed because `IllegalArgumentException` is thrown elsewhere in the code